tests/readme-icons.test.js

```diff
@@ -1,34 +1,44 @@
-import { promises as fs } from 'node:fs';
+import fs from 'node:fs';
 import path from 'node:path';
 import { fileURLToPath } from 'node:url';
-import { test, exec } from 'uvu';
+import { test } from 'uvu';
 import * as assert from 'uvu/assert';
 
-(async () => {
-  const __dirname = path.dirname(fileURLToPath(import.meta.url)),
-    root = path.dirname(__dirname),
-    darkIconsPath = path.join(root, 'icons'),
-    lightIconsPath = path.join(root, 'assets', 'readme'),
-    lightIconsFileNames = await fs.readdir(lightIconsPath);
+const __dirname = path.dirname(fileURLToPath(import.meta.url)),
+  root = path.dirname(__dirname),
+  blackIconsPath = path.join(root, 'icons'),
+  whiteIconsPath = path.join(root, 'assets', 'readme'),
+  whiteIconsFileNames = fs.readdirSync(whiteIconsPath);
```

Comment:
Personally, I prefer a declaration for each variable rather than multiple at the same time. Do you agree?

Reply:
I have no preference, done in https://github.com/simple-icons/simple-icons/pull/7056/commits/e551891e8d0f7d01f453d54675e5078e1f19deb6